WordPress/twentytwentyfive
# Fix: Move link in bio (single page) patterns out of "About"

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/307

Moving the link in bio "pages" template out of the "About" category.

**Screenshots**



https://github.com/user-attachments/assets/6747f37d-65d0-4a71-b87d-ced0e5ae21f4

**Testing Instructions**

1. Create a page.
2. Try adding a pattern from the block inserter
3. Confirm that the three patterns are outside the "About" category.
